virtme_ng/version.py

```diff
@@ -4,10 +4,9 @@
 """virtme-ng version"""
 
 import os
+from importlib.metadata import PackageNotFoundError, version
 from subprocess import DEVNULL, CalledProcessError, check_output
 
-from importlib.metadata import version, PackageNotFoundError
```

Comment:
We should really have the CI checking that to avoid such commit I think. (or at least to tell us a fix will be needed).

@arighi do you want to try https://pre-commit.ci? It will suggest the autofix we can squash after if needed. From what I got, you simply need to login from this page and accept the integration for this repo.

Or do you prefer the "[lite](https://pre-commit.ci/lite.html)" integration in a new GitHub action workflow? Or nothing?